XEVIOUS/GameEnemy.cpp

```diff
@@ -0,0 +1,152 @@
+﻿#include"GameEnemy.h"
+
+
+GameEnemy::GameEnemy()
+{
+	m_pDirectX = DirectX::GetInstance();
+}
+
+GameEnemy::~GameEnemy()
+{
+
+}
+
+
+void GameEnemy::Update()
+{
+	MoveOperation();
+}
+
+void GameEnemy::Render()
+{
+	for (auto& centerEnemy : m_EnemyCenters)
+	{
+		if (centerEnemy.AppearCount >= centerEnemy.AppearTime * 60)
+		{
+			m_pDirectX->DrawTexture("GAME_ENEMY_TEX", centerEnemy.Enemy);
+		}
+		centerEnemy.AppearCount++;
+	}
+}
+
+void GameEnemy::MoveOperation()
+{
+	for (auto& centerEnemy : m_EnemyCenters)
+	{
+		D3DXVECTOR2 moveDirection(0.f, 0.f);
+		if (centerEnemy.AppearCount >= centerEnemy.AppearTime * 60)
+		{
+			centerEnemy.MovementChangeCount++;
+			switch (centerEnemy.MovePattern)
+			{
+			case NORMAL:
+				moveDirection.y += 1.f;
+				ENEMY_MOVE_SPEED = 4.f;
```

Comment:
```suggestion
				enemySpeed = 4.f;
```
constと紛らわしいので変数名を修正したほうがいいと思います
また速さが全エネミー共通なので値を変える必要はないのでは

Reply:
もしconstにしたらエネミーによって動く速さを変えたい場合どうするん？

Reply:
ごめんこの質問なしで

Reply:
const じゃない変数を大文字で書かないようにしようということですね

Reply:
おけおけ